docs/07_MCP_SERVER.md

```
@@ -1,627 +1,212 @@
-# Send JSON-RPC request
-echo '{"jsonrpc":"2.0","id":1,"method":"tools/list"}' | node apps/mcp/dist/stdio.js
-
-# Expected output
-{
-  "jsonrpc": "2.0",
-  "id": 1,
-  "result": {
-    "tools": [...]
-  }
-}
+$ echo '{"id": "1", "method": "tools/list"}' | pnpm --filter @synthesis/mcp dev
```

markdownlint-cli2 (0.18.1)

Dollar signs used before commands without showing output

(MD014, commands-show-output)